src/lib/component/partial/SigninDialog/SigninForm.js

```diff
@@ -14,77 +14,71 @@ const StyledDialoagActions = createStyledComponent({
   }),
 });
 
-class SignInForm extends React.Component {
-  static propTypes = {
-    disabled: PropTypes.bool,
-    error: PropTypes.string,
-    onSubmit: PropTypes.func.isRequired,
-  };
+const SignInForm = ({ disabled, error, onSubmit }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
 
-  static defaultProps = {
-    disabled: false,
-    error: null,
-  };
+  const handleSubmit = useCallback(
+    ev => {
+      onSubmit({ username, password });
+      ev.preventDefault();
+    },
+    [username, password],
```

Comment:
onSubmit should be included in the deps array

Reply:
Good eye 👀